www/src/util/config.ts

```diff
@@ -1,13 +1,22 @@
 const AUTH0_DOMAIN = process.env.REACT_APP_AUTH0_DOMAIN || '';
 const AUTH0_CLIENT_ID = process.env.REACT_APP_AUTH0_CLIENT_ID || '';
 
+const SERVER_ENDPOINT = process.env.REACT_APP_SERVER_ENDPOINT || '';
+const SERVER_AUDIENCE = process.env.REACT_APP_SERVER_AUDIENCE || '';
```

Comment:
IMO these should have sane defaults for local development e.g. `const SERVER_ENDPOINT = process.env.REACT_APP_SERVER_ENDPOINT || 'http://localhost:1337';`